src/Cartography.DynamicMap/GeoLocatedGrouping.cs

```diff
@@ -2,25 +2,24 @@
 using System.Collections.Generic;
 using System.Linq;
 using Uno.Extensions;
-using Windows.Devices.Geolocation;
 
 namespace Cartography.DynamicMap;
 
 public class GeoLocatedGrouping<T> : IGeoLocatedGrouping<T>
 {
-	private IGrouping<Geocoordinate, T> _source;
+	private IGrouping<BasicGeoposition, T> _source;
```

Comment:
Try to keep the same API surface. I suggest to use a class named Geocoordinate like before.

Reply:
Oh right I forgot to revert!